Makefile

```diff
@@ -78,7 +78,7 @@ checks: lint go-mod-tidy
 
 generate:
 	$(CONTROLLER_GEN) object:headerFile="build/boilerplate/boilerplate.go.txt" paths="./pkg/apis/..."
-	$(CONTROLLER_GEN) crd:crdVersions=v1beta1,trivialVersions=true \
+	$(CONTROLLER_GEN) crd:crdVersions=v1,trivialVersions=true \
 		paths="./pkg/apis/..." \
 		output:crd:artifacts:config=deployments/operator/crd/bases
 	$(CONTROLLER_GEN) crd:crdVersions=v1beta1,trivialVersions=true \
```

Comment:
@mythi Should we do the same for fpga_admissionwebhook?

Since the webhook is a part of the operator now I wonder if we better simply drop it at some point. IDK...

Reply:
> @mythi Should we do the same for fpga_admissionwebhook?

I had missed we had two copies of those. Did you just copy them last time?

> Since the webhook is a part of the operator now I wonder if we better simply drop it at some point. IDK...

It's a valid question and deserves discussion.

Reply:
> Did you just copy them last time?

Yes, in a sense. Two different sets of deployment YAMLs are generated for the operator and for the webhook. The operator YAMLs comprise everything `controller-gen` can find inside `./pkg/apis`. The webhook ones includes only the subset relevant to the webhook - from `./pkg/apis/fpga`.

Reply:
@rojkov AFAICS se CRDs are generated once but maintained in two places. Sounds like we should add something like `cp deployments/operator/crd/bases/fpga.intel.com* deployments/fpga_admissionwebhook/crd/bases/` in `make generate`. WDYT?

Reply:
I see it a bit differently: the sources for the CRDs are maintained in one place, but the CRDs are generated twice. If it's guaranteed that the generated CRDs are going to be equal always (there's no dependency on broader contexts or target paths) then `cp`ing instead of generating is a no-brainer. 

Originally the FPGA webhook was based on a more low-level controller lib from K8s. Later I unified it with the operator by rewriting it to use kubebuilder to reduce maintenance burden. But they are still independent and dropping one of them won't interfere with the other. I'm afraid "generate once then copy" can create a quantum entanglement between them :)

I'd rather factor out `v1` to a variable and use it in the two `controller-gen` runs.

Reply:
> I'd rather factor out `v1` to a variable and use it in the two `controller-gen` runs.

Scratch my stupid comments, I had completely missed the fpga_webhook CRDs have their own `controller-gen` run. I agree with your proposal.